test/helpers/bitcoin-core.ts

```diff
@@ -33,6 +33,26 @@ export async function sendToAddressAndConfirm({
   await walletClient.generateToAddress({ nblocks: 6, address: RANDOM_ADDRESS })
 }
 
+export async function sendToAddress({
+  walletClient,
+  address,
+  amount,
+}: {
+  walletClient: BitcoindWalletClient
+  address: OnChainAddress
+  amount: number
+}) {
+  await walletClient.sendToAddress({ address, amount })
```

Comment:
just return, await is not needed

Reply:
Done: https://github.com/GaloyMoney/galoy/pull/1511/commits/423357013149bcde54e0310937d831695722ded4